src/behavior-considered-undefined.md

```diff
@@ -51,7 +51,15 @@ code.
 * Invoking undefined behavior via compiler intrinsics.
 * Executing code compiled with platform features that the current platform
   does not support (see [`target_feature`]), *except* if the platform explicitly documents this to be safe.
-* Calling a function with the wrong call ABI or unwinding from a function with the wrong unwind ABI.
+* Calling a function with the wrong call ABI or unwinding from a function with the wrong unwind ABI
+* Calling a foreign (e.g. C++) function that unwinds (`throw`s) via a function
+  declaration or pointer declared with a non-unwinding ABI such as `"C"`
+* Calling a Rust `extern` function that unwinds (with `extern "C-unwind"` or
+  another ABI that permits unwinding) from a runtime that does not support
+  unwinding, such as code compiled with GCC or Clang using `-fno-exceptions`
+* Deallocating a Rust stack frame without executing destructors
+  for local variables owned by the stack frame. This can occur
+  with C functions like `longjmp`.
```

Comment:
Should this section includes "Catching foreign unwinding using `catch_unwind`" and `Catching Rust unwinding using foreign catch mechanisms" as UB? [Rustonomicon says it's UB](https://doc.rust-lang.org/stable/nomicon/ffi.html#ffi-and-unwinding) but I couldn't find any mention of it in this PR.

Reply:
Thanks! I thought that was already there. Good catch.